oak-run/src/main/java/org/apache/jackrabbit/oak/run/DataStoreCheckCommand.java

```diff
@@ -188,9 +188,9 @@ static int checkDataStore(String... args) {
             NodeStore nodeStore = null;
             if (options.has(store)) {
                 String source = options.valueOf(store);
-                if (source.startsWith(MongoURI.MONGODB_PREFIX)) {
-                    MongoClientURI uri = new MongoClientURI(source);
-                    MongoClient client = new MongoClient(uri);
+                if (source.startsWith(MongoConnection.MONGODB_PREFIX)) {
```

Comment:
Implementation of `options.valueOf(store)` clearly states that it can return an null value if it returns an `emptyList()`. I think we should handle the null check here as well.